src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1624,7 +1677,7 @@ namespace FanCoilUnits {
 				// get full load result
 				mdot = FanCoil( FanCoilNum ).MaxColdWaterFlow;
 				SetComponentFlowRate( mdot, FanCoil( FanCoilNum ).ColdControlNode, FanCoil( FanCoilNum ).ColdPlantOutletNode, FanCoil( FanCoilNum ).CWLoopNum, FanCoil( FanCoilNum ).CWLoopSide, FanCoil( FanCoilNum ).CWBranchNum, FanCoil( FanCoilNum ).CWCompNum );
-				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMaxHC );
+				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMaxHC, _, ElectricHeaterControl );
```

Comment:
The electric heating coil is tricky in that it will turn on to meet the load. In this case, a cooling load is present and the heating coil must be off. Had to add an argument for the electric heating coil to turn it on and off as needed.